db/migrate/20171204104811_create_metadata.rb

```diff
@@ -0,0 +1,19 @@
+class CreateMetadata < ActiveRecord::Migration[5.1]
+  def change
+    create_table :metadata do |t|
+      t.string :category
+      t.string :resource
+      t.string :version
+      t.string :contact_organisation
+      t.string :dataset_id
+      t.string :metadata
+      t.string :factsheet
+      t.boolean :marine_spatial_planning
+      t.boolean :education
+      t.boolean :environmental_impact_assessment
+      t.boolean :ecosystem_assessment
+      t.boolean :ecosystem_services
+
```

Comment:
Can we please add the timestamps as well?

Reply:
Sorry @Levia I will update this immediately!